k8s/agent/deploy-agent.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tracetest-agent
+  labels:
+    app: tracetest-agent
+spec:
+  selector:
+    matchLabels:
+      app: tracetest-agent
+  template:
+    metadata:
+      labels:
+        app: tracetest-agent
+    spec:
+      containers:
+        - name: tracetest-agent
+          image: kubeshop/tracetest-agent:v0.13.9
```

Comment:
Should we use `latest` here?

Reply:
It may be a good idea. I believe that it is easy to forget to update this version.